app.ts

```diff
@@ -400,7 +404,7 @@ function startListening(server) {
     if (ss) {
         // ms (5 min default)
         const idleTimeout = process.env.IDLE_TIMEOUT;
-        const timeout = (idleTimeout === undefined ? 300 : idleTimeout) * 1000;
+        const timeout = (idleTimeout === undefined ? 300 : parseInt(idleTimeout)) * 1000;
```

Comment:
Ts thinks `process.env.IDLE_TIMEOUT` is a `string|undefined` and I think it's probably right

Reply:
Yeah, all of process.env is a Record<string, string | undefined>